runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -422,6 +428,14 @@ protected DataflowRunner(DataflowPipelineOptions options) {
       // Dataflow Streaming runner overrides the SPLITTABLE_PROCESS_KEYED transform
       // natively in the Dataflow service.
     } else {
+      overridesBuilder
```

Comment:
Can you extract whole if/else or contents of if/else cases into separate methods please? First if case is pretty big to be worth moving to separate method/class.

Reply:
Breaking up the contents will hinder understanding what happens in this method since many of these transforms have to be overridden in a specific order so that X is replaced before the Y replacement happens since X replaces part of something that then Y replaces to be something else.

Reply:
Makes sense.